src/components/issues/ActionForm.tsx

```diff
@@ -135,10 +137,18 @@ const ActionForm: FC<ActionFormProps> = ({
                                 : styles.suggestions_container
                         }
                     >
+                        <label
+                            htmlFor="assignee"
+                            className={styles.assign_label}
```

Comment:
Is className `assign_label` intended or a typo?

Reply:
Intended it has been used in several places in several different complements already.